### PR TITLE
Help ui cleanup

### PR DESCRIPTION
**Pull request by reginapramesti**
Just a bunch of tidying up - fix minor errors, remove console logs, reimplement things that were a bit hacky

### CODE REVIEW

**Comment by soyarsauce** [comment]
Restarted CI job

**Comment by soyarsauce** [comment]
Restarted CI job

File: lib/Styled/Text.jsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const Text = styled.div`
     font-size: 15px;
   `}
   ${props =>
-    props.title &&
+    props.heading &&
```
Comment (soyarsauce) [inline lib/Styled/Text.jsx:51]: @reginapramesti have a look at what I've done in
https://github.com/TerriaJS/terrace/blob/master/lib/Styled/Heading.tsx#L20

Maybe it's time we brought that across, it's basically a wrapper around `Text` but with the proper html element for whichever heading we want to use, I'm not sure what the final solution/mishmash is going to look like, let me know what you think 🙂

**Review by soyarsauce — APPROVED** [review]

File: lib/Styled/Text.jsx (hunk 1 above)
Comment (reginapramesti) [reply lib/Styled/Text.jsx:51]: @soyarsauce oooh yeah that would be really good to have I reckon!

**Pull request merged**